crates/biome_css_analyze/tests/specs/nursery/noDuplicateSelectors/valid.css

```diff
@@ -1,34 +1,33 @@
-/* Same selector with different parent nodes are allowed */
-.foo {
-    width: 12;
-}
-
-@media (min-width: 10px) {
-    .foo {
-        width: 12;
-    }
-}
-
-.bar {
-    .bar {
-        width: 12;
-    }
-}
-
-/* Same selector in a different list */
-.foo .bar {
-    width: 12;
-}
-
-/* Net list is different */
-a b {
-    width: 12;
-}
-
-a {
-
-    & b,
-    & c {
-        width: 12;
-    }
-}
\ No newline at end of file
+/* no duplicates */
+a {} b {} c {} d, e, f {}
+
+/* duplicate inside media query */
+g {}
+@media print { g {} }	
+
+/* duplicate inside keyframes */
+@keyframes h { 0% {} }
+@keyframes h { 0% {} }
+
+/* duplicates inside nested rules */
+i { i { i {} } }
+
+/* selectors using parts of other selectors */
+.foo .bar {}
+.foo {}
+.bar {}
+.bar .foo {}
+
+/* selectors reused in other non-equivalent selector lists */
+j {} j, k {}
```

Comment:
minor

Can you add the equivalent j {} k, j {}?

Reply:
Added an equivalent below, thank you for the recommendation! 🙇 
https://github.com/biomejs/biome/pull/2660/commits/3d58e8e2aea2f6dff41b03d459c51862f089dc0d#diff-6801695a7b652ea030739d5435647592ad00059de63b771a6413b7f622c6df9dR23